### ISSUE TITLE
USER ISSUE: Server Crash

### ISSUE THREAD

**Issue by geggleto** [StrangeLoopGames/EcoIssues#22577]
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/7041773/Player.log)
[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/7041775/Player-prev.log)

**User Id: (Not logged user)**

**Version:** 0.9.4.0 beta release-237

**Steps to Reproduce:**
No idea.

**Expected behavior:**

**Actual behavior:**
The following error message popped up:
```
Server encountered an exception:

<size=60.00%>Exception: NullReferenceException
Message:Object reference not set to an instance of an object.
Source:System.Private.CoreLib

System.NullReferenceException: Object reference not set to an instance of an object.
   at System.Object.GetType()
   at Eco.Shared.Networking.RPCManager.TryCreateInvocation[TArg](INetClient client, Object target, String methodName, IList`1 packedArgs, Func`3 unpackArg, RPCInvocation& invocation) in D:\jenkins_agent\workspace\Eco Release\Server\Eco.Shared\Networking\RPCManager.cs:line 422
   at Eco.Core.Controller.ControllerManager.HandleViewRPC(INetClient client, Int32 controllerID, String methodName, BSONObject bson) in D:\jenkins_agent\workspace\Eco Release\Server\Eco.Core\Controller\ControllerManager.cs:line 736
Outer Exceptions:
Exception has been thrown by the target of an invocation.</size>
```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
None this is a hosted world

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by Daniel-Magulado** [comment]
Hi @geggleto 

Thank you for writing this bug report.
Can you share with us your complete operating system version? how often do you encounter this issue, and is there any specific scenarios where once done, the game crashes for you frequently?
Is this a migrated or a new world?
Thank you.

**Comment by geggleto** [comment]
Windows 10

First time experiencing this. I have noticed that the Eco World server has some quirks might be releated?

I was adding labour to a campfire job.

This is a new-world, 3-4 days old.

The Eco World had some odd bugs; it was showing 1 user as having 150+ rooms. After a reboot it went back to normal.

After I rebooted the server it was fine.

to be clear the server is an Eco World Server Instance.
